sass/site/primary/_posts-and-pages.scss

```diff
@@ -222,3 +222,41 @@
 	margin-bottom: 50px;
 	border-bottom: 2px solid $color__borders;
 }
+
+body.search-results .content-area {
```

Comment:
In the past we've had feedback from the guys at [themereview.co](http://themereview.co/) tell us that it's bad practice to use a `body.xxxx` selector. Ideally we should avoid it, unless it's important for specificity or where we're worried about class name conflicts. In this case, just `.search-results` should be fine.